### ISSUE TITLE
Header.js throws error with @shoutem.theme related

### ISSUE THREAD

**Issue by bankgg** [GeekyAnts/NativeBase#485]
"native-base": "^2.0.3",
"react": "15.4.2",
"react-native": "0.41.2"

Expected behavior
- The app shows the basic components perfectly.

Actual behavior
- The app throws an error somewhat related to @shoutem.theme as attached.

Steps to reproduce
- Just put a simple code that includes <Header></Header> tag to the render() method like this:

```
render() {
    return (
      <Container>
                <Header>

                </Header>
      </Container>
    );
}
```

Screenshot
- Link: [http://imgur.com/a/Jd0eG](http://imgur.com/a/Jd0eG)

The app is tested only on an Android device (Samsung Galaxy S6 edge 6.0.1).

**Comment by omerdn1** [comment]
Same here with the new version (Upgrading from 0.5.13 to 2.0.3)

**Comment by sankhadeeproy007** [comment]
Hi, can you just update to `2.0.4` and check if it works now?

**Comment by zdwebber** [comment]
I no longer have this issue after upgrading to 2.0.4

**Comment by bankgg** [comment]
It's solved now with 2.0.4.
